### DISCUSSION TITLE
adding a repo

### DISCUSSION THREAD

**Discussion by helsingi** [taishi-i/awesome-ChatGPT-repositories#53] [General]
hi, I am not sure I understand how to suggest a new repo to be added, hopefully it's ok to do that here:
https://github.com/reorproject/reor
Thanks for the awesome list!

**Comment by taishi-i** [comment]
Thank you for your suggestion, and I apologize for the late reply. I will add this repository in our next update. Thank you again for sharing the information.

**Comment by taishi-i** [comment]
The addition has been completed and the multilingualization of information has been finalized. Thank you! 

https://github.com/taishi-i/awesome-ChatGPT-repositories/blob/f5f45d7f2bd1b66e9b70b657fe93a497a3b9dafb/README.md?plain=1#L29
